### DISCUSSION TITLE
Add tooltip API for all UI controls

### DISCUSSION THREAD

**Discussion by AbbottMc** [Mojang/minecraft-editor#177] [Feedback]
If possible, please add tooltip api (like the image shows below) for all UI controls so that we can provide a in game guide for users and low down the learning cost. 

![image](https://github.com/Mojang/minecraft-editor/assets/16969459/2ec1fca6-ada4-4463-9d9e-0cbed01959bf)

For example, I got a input field here and when users hover the cursor on the field, it will show the tooltip that introducts how user can use this param to adjust the generation behavior or something. Compare to the out-game documentations, it will provide a more smooth use experience.

![image](https://github.com/Mojang/minecraft-editor/assets/16969459/054f8ba0-85f7-46e7-b342-70950c18f05c)

Currently we can only add tooltips for Tool tab. QAQ

**Comment by dacowan** [comment]
Totally agree -- adding Tool Tips for all UI controls is already on our backlog

**Reply by AbbottMc** [reply]
Hooray!!

**Reply by dacowan** [reply]
We've started rolling out new bindings for the UI controls - all of the new bindings (not the ones marked _deprecated) - now have a tooltip field.
